src/js/providers/html5.js

```diff
@@ -11,8 +11,8 @@ define([
 ], function(cssUtils, utils, stretchUtils, _, events, states, eventdispatcher, DefaultProvider, video) {
 
     var clearInterval = window.clearInterval,
-        stallInterval,
         STALL_DELAY = 256,
+        BUFFER_INTERVAL = 50,
```

Comment:
Did you switch this from 100 for a specific reason? The less often we check the better imho